### PR TITLE
Bug fix/small issues 21022019

### PR DESCRIPTION
**Pull request by jsteemann**

### CODE REVIEW

**Comment by jsteemann** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/3137/

File: arangod/RocksDBEngine/RocksDBBuilderIndex.cpp [hunk 1]
```diff
@@ -243,6 +243,10 @@ static arangodb::Result fillIndex(RocksDBIndex& ridx, WriteBatchType& batch,
 
   if (res.ok()) {  // required so iresearch commits
     res = trx.commit();
+
+    if (ridx.estimator() != nullptr) {
+      ridx.estimator()->setCommitSeq(rootDB->GetLatestSequenceNumber());
```
Comment (graetzer) [inline arangod/RocksDBEngine/RocksDBBuilderIndex.cpp]: you forgot the catchup code

**Comment by jsteemann** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/3168/

**Comment by jsteemann** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/3197/

**Comment by jsteemann** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/3235/

**Pull request merged**